services/ui-src/src/components/report/StatusTable.tsx

```diff
@@ -41,23 +39,17 @@ export const StatusTableElement = () => {
     return (
       <Tr key={section.id || index} p={0}>
         <Td>
-          <Stack flex="1">
-            <Text fontWeight="bold">{section.title}</Text>
-          </Stack>
+          <Text>{section.title}</Text>
         </Td>
         <Td>
-          <Flex align="right">
-            {/* TODO: Logic for when a page is incomplete to change status icon and text */}
-            <Image
-              src={iconStatusCheck ? iconStatusCheck : iconStatusError}
-              alt="icon description"
-            />
-            <Text ml={1}>{iconStatusCheck ? "Complete" : "Error"}</Text>
-          </Flex>
+          {/* TODO: Logic for when a page is incomplete to change status icon and text */}
+          <TableStatusIcon
+            tableStatus={"complete"}
+            isPdf={true}
```

Comment:
Just curious, what is `pdf`? It's not like a PDF file right? It's the alt text for the icons?

Reply:
it's a leftover from when i migrated this from mfp. we reuse the same components for when we build the pdf so i'm currently not sure whether to remove it or not since we don't have a pdf page to refer to. i'm leaving it in for now until i have to remove it.